src/control/system/member.go

```diff
@@ -36,13 +36,18 @@ const (
 	MemberStateStopping MemberState = 0x0010
 	// MemberStateStopped indicates process has been stopped.
 	MemberStateStopped MemberState = 0x0020
-	// MemberStateEvicted indicates rank has been evicted from DAOS system.
+	// MemberStateEvicted indicates rank has been automatically evicted from DAOS system.
 	MemberStateEvicted MemberState = 0x0040
+	// MemberStateExcluded indicates that the rank has been administratively excluded.
+	MemberStateExcluded MemberState = 0x0080
 	// MemberStateErrored indicates the process stopped with errors.
-	MemberStateErrored MemberState = 0x0080
+	MemberStateErrored MemberState = 0x0100
 	// MemberStateUnresponsive indicates the process is not responding.
```

Comment:
Is it OK to shift these enums at this point, or would it be better to tack onto the end of the list? Since the DB is saved as metadata, we could read it back up after an upgrade understanding all the states incorrectly (assuming MemberState is saved, anyway).

Reply:
Hmm. Good catch. It would be safer to append rather than reorder at this point.